README.md

```diff
@@ -1752,6 +1752,7 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 
 - [cState](https://github.com/cstate/cstate/) - Static status page for hyperfast Hugo. Clean design, minimal JS, super light HTML/CSS, high customization, optional admin panel, read-only API, IE8+. Best used with Netlify, Docker. ([Demo](https://cstate.mnts.lt/)) `MIT` `Go`
 - [s.Status](https://github.com/scolastico-dev/s.Status) - s.Status is a open source server status page written in java. ([Demo](https://status.scolasti.co/)) `MPL-2.0` `Java`
+- [Uptime Kuma](https://github.com/louislam/uptime-kuma) - A self-hosted website monitoring tool like "Uptime Robot". ([Demo](https://demo.uptime.kuma.pet)) `MIT` `JavaScript`
```

Comment:
> npm run

>  The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.

JavaScript > Nodejs

Reply:
Node.js is an interpreter, JavaScript is a language, I'm not clear on what needs to change?

Reply:
> Node.js is an interpreter, JavaScript is a language, I'm not clear on what needs to change?

>> https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/.github/PULL_REQUEST_TEMPLATE.md:
>> The Language tag is the main server-side requirement for the software

`Javascript` is for static HTML5/JS applications. Programs that need Nodejs server-side must be labeled `Nodejs`

Reply:
ping @builder555 can you fix the language/platform tag?